mne/viz/topomap.py

```diff
@@ -1051,10 +1051,12 @@ def plot_evoked_topomap(evoked, times=None, ch_type=None, layout=None,
         raise RuntimeError('Too many plots requested. Please pass fewer '
                            'than 20 time instants.')
     tmin, tmax = evoked.times[[0, -1]]
-    for t in times:
-        if not tmin <= t <= tmax:
-            raise ValueError('Times should be between %0.3f and %0.3f. (Got '
-                             '%0.3f).' % (tmin, tmax, t))
+    _time_comp = _time_mask(times=times, tmin=tmin,  tmax=tmax)
+    if not np.all(_time_comp):
+        raise ValueError('Times should be between {0:0.3f} and {1:0.3f}. (Got '
+                         '{2}).'.format(
+                             tmin, tmax,
+                             ['%03.f' % t for t in times[_time_comp]]))
```

Comment:
Fine by me, FYI you can also use `np.array_str(x, None, 3)` for this purpose but it won't insert zeros like your current formatting does so it's probably not as good